### DISCUSSION TITLE
Clear HSTS entry for specific site

### DISCUSSION THREAD

**Discussion by yogo1212** [qutebrowser/qutebrowser#6244] [Q&A] [answered]
Hi :-)

To access an opnsense installation remotely, I forwarded the UI via SSH and got myself a HSTS entry for localhost:8080.
Is there a/any way to get rid off it?

(can't find any settings related to HSTS / `git grep -i hsts`)

regards!

**Comment by The-Compiler** [comment, marked as answer]
I don't think this is possible, unfortunately. It's entirely handled by the underlying QtWebEngine/Chromium, and the [way you'd do this in Chrome](https://www.thesslstore.com/blog/clear-hsts-settings-chrome-firefox/) unfortunately isn't accessible with QtWebEngine.

The underlying data is stored in `~/.local/share/qutebrowser/webengine/TransportSecurity` as far as I know - it's a JSON file, but for some reason, the hosts are some base64 binary, so I'm not sure how the hostnames are actually stored in there...

I've opened a feature request for QtWebEngine to provide a way to do this: [[QTBUG-91695] Expose chrome://net-internals or API to clear HSTS entries - Qt Bug Tracker](https://bugreports.qt.io/browse/QTBUG-91695)

**Reply by yogo1212** [reply]
@The-Compiler thanks for the info :-)

**Reply by The-Compiler** [reply]
Looks like the `chrome://net-internals` page is getting added to QtWebEngine: [Add web-ui chrome://net-internals (Ie00b9bb9) · Gerrit Code Review](https://codereview.qt-project.org/c/qt/qtwebengine/+/338140) - seems to work fine for me, so this will probably be in QtWebEngine 5.14.4. :slightly_smiling_face:
